src/components/Articles.jsx

```diff
@@ -0,0 +1,40 @@
+import { useEffect, useState } from "react";
+import { getArticles } from "../api";
+import ArticleCard from "./ArticleCard";
+
+const Articles = () => {
+    const [articles, setArticles] = useState([]);
+
+    useEffect(() => {
+        getArticles().then((res) => {
+            setArticles(res.articles)
+        })
```

Comment:
some loading state would be nice here so that users are informed the request is ongoing